### PR TITLE
Fix/testing and add contact bugs

### PR DESCRIPTION
**Pull request by carolineshowscode**
## Type of Change
- [ ] JSX
- [ ] HTML
- [ ] JavaScript
- [ ] CSS
- [x] React
- [ ] React Router
- [x] Cypress
- [ ] Apollo
- [ ] GraphQL
- [ ] Twilio API
- [x] Refactor/Fix Bugs

## Please give a short description of the changes that were made below

- Fix:
- Cypress sad path timing
- Cypress `Alert` sad path

- Feature:
- `Readme` description and outline
- `AboutUs` detailed project/concept description

## How has this been tested?
- [x] In Browser
- [ ] In Console
- [x] Cypress
- [ ] Postman

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,106 @@
...
+* The application is deployed to Heroku, to view the live application: https://walk-safe-frontend.herokuapp.com/
+
+* To view the application on your local device:
+
+1. Clone down this application
+2. Enter `npm i` in your terminal
+3. Enter `npm start` in your terminal
+4. To view cypress tests open separate tab an enter `npx cypress run`
+
+## Front End Authors
```
Comment (bhohnco) [inline README.md:50]: Solid work on this, looks great!

File: cypress/integration/sad-paths.js [hunk 2]
```diff
@@ -14,38 +14,73 @@ beforeEach(() => {
...
+            .wait(200)
+          .get('.submit-trip-btn')
+            .click()
+          .get('.begin-trip-btn')
+            .click()
+          .get('.trip-container').should('be.visible')
+          .get('.mm-popup__box')
+          .get('.mm-popup__btn--ok').click()
+          .wait(58000)
+          .get('.add-time-modal').should('be.visible')
```
Comment (bhohnco) [inline cypress/integration/sad-paths.js:78]: This is such a long test, great job putting it all together!

**Review by bhohnco — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,106 @@
...
+* [Learning Goals](#LearningGoals)
+* [Installation](#installation)
+* [Walkthrough](#walkthrough)
+* [Authors](#authors)
+* [Technologies](#technologies)
+* [Demos](#demos)
+
+## Description
+
+Created to empower individuals to walk alone, Walk Safe brings a user's community along from the trip. Utilizing Geolocation and SMS messaging APIs, we keep a user's community informed of their trip status from start to finish. Should an unforeseen trip interruption arise, Walk Safe will immediately alert the user's contact. We've got your back.
```
Comment (pcmueller) [inline README.md:15]: Well phrased!

File: README.md [hunk 4]
```diff
@@ -1 +1,106 @@
...
+## Learning Goals
+
+
+
+*Minimum Viable Product (MVP)*:  
+
+
+## Notable Features
+
+- GoogleMaps Places API to autocomplete start and end point address
```
Comment (pcmueller) [inline README.md:30]: This reads very professional!

File: README.md [hunk 5]
```diff
@@ -1 +1,106 @@
...
+    </tr>
+    <tr>
+        <td><img src="" alt="P. Muellerleile" width="125" height="auto" /></td>
+    </tr>
+</table>
+
+## Technologies
+<table>
+    <tr>
+        <td>JavaScript</td>
```
Comment (pcmueller) [inline README.md:77]: Nice, love the inclusion of tech logos.  If you're missing any I've got a bunch more - Hooks, TravisCI, etc.  Do we want to include BE tech here as well?

**Review by pcmueller — APPROVED** [review]
Thank you for doing this, happy to help fill in any gaps!

**Pull request merged**